index.d.ts

```diff
@@ -6,7 +6,7 @@ export type ThunkAction<R, S, E> = (dispatch: Dispatch<S>, getState: () => S,
 
 declare module "redux" {
   export interface Dispatch<S> {
-    <R, E>(asyncAction: ThunkAction<R, S, E>): R;
+    <R, E>(asyncAction: ThunkAction<R, S, E> | S): R;
```

Comment:
Why would `asyncAction` have the type of the state (i.e. `S`)?

Reply:
Yep, shouldn't this be `Action`. From `import { Action } from 'redux'`?